tests/bluetooth/bsim_bt/bsim_test_eatt/prj_gh_42306.conf

```diff
@@ -0,0 +1,20 @@
+CONFIG_BT=y
+CONFIG_BT_CENTRAL=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_SMP=y
+CONFIG_BT_BONDABLE=y
+CONFIG_BT_ATT_PREPARE_COUNT=8
+CONFIG_BT_GATT_CLIENT=y
+CONFIG_BT_GATT_DYNAMIC_DB=y
```

Comment:
Do we need these two configs?

Reply:
No, for this test they can be removed. The test here is not up to date with the L2CAP collision mitigation PR.